app/concerns/long_runnable.rb

```diff
@@ -117,6 +121,7 @@ def boolify(value)
     end
 
     private
+
```

Comment:
Haha, I don't feel strongly. Well, except apparently I do enough to keep changing it :)